pkg/api/types.go

```diff
@@ -5,6 +5,10 @@ import docker "github.com/fsouza/go-dockerclient"
 const (
 	DefaultNamespace    = "io.openshift.s2i."
 	KubernetesNamespace = "io.k8s."
+	DefaultIgnoreFile   = ".s2iignore"
+	DefaultUploadSubdir = "upload"
+	DefaultSrcSubdir = "src"
```

Comment:
OK, I've updated the constants in `scripts.go` to leverage os.PathSeparator and verified in go playground, as well as test-go.sh and test-integration.sh.  Will push in a couple.